### PR TITLE
[Overflow-452] Refactor layout

### PR DESCRIPTION
**Pull request by jescabillas**
## Backlog Link
https://framgiaph.backlog.com/view/SUN_OVERFLOW-452

## Commands
None

## Pre-conditions
None

## Expected Output
The layout will follow the redesign navbar, left and right sidebar components.

## Notes
The screenshot is a bit off since it is using the old design.

## Screenshots
image

### CODE REVIEW

**Review by nicoleamber — COMMENTED** [review]
Instead of specifying the height / padding / etc in pixels, you can refer to tailwind documentation and use their respective classes.

For example: `h-[56px]` => `h-14` (reference: https://tailwindcss.com/docs/height)

**Comment by jescabillas** [comment]
> Instead of specifying the height / padding / etc in pixels, you can refer to tailwind documentation and use their respective classes.
> 
> For example: `h-[56px]` => `h-14` (reference: https://tailwindcss.com/docs/height)

This has been adjusted

**Review by nicoleamber — DISMISSED** [review]

File: frontend/components/templates/layouts/TopLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import Navbar from '@/components/organisms/Navbar'
+
+type Props = {
+    children: JSX.Element
+}
+
+const TopLayout = ({ children }: Props): JSX.Element => {
+    return (
+        <div className="relative h-screen">
+            <div className="fixed z-20 h-14 w-full border-b-2 border-neutral-200 drop-shadow-[0_2px_2px_rgba(0,0,0,0.05)]">
```
Comment (chevyB) [inline frontend/components/templates/layouts/TopLayout.tsx]: Drop shadow is repetitive. You can create a config for this.
https://tailwindcss.com/docs/drop-shadow#customizing-your-theme

**Review by kenorenzsun — DISMISSED** [review]

**Review by ipilikent — DISMISSED** [review]

File: frontend/components/templates/layouts/TopLayout.tsx (hunk 1 above)
Comment (jescabillas) [reply frontend/components/templates/layouts/TopLayout.tsx]: This has been applied in the latest commut

**Review by kenorenzsun — APPROVED** [review]

**Review by nicoleamber — APPROVED** [review]

**Review by chevyB — APPROVED** [review]

**Review by ipilikent — APPROVED** [review]

**Comment by ipilikent** [comment]
## Approved PR
- The layout for navbar, left and right sidebar components have changed


## Proof
<details>

352 -Refactor Layout

</details>

-----
- Total Test time:  0.25 hr.
- Total Comments: 0
- Total Iterations: 0

**Pull request merged**